Configuration/QualificationTokens.json

```diff
@@ -684,7 +684,7 @@
   },
     {
     "accessToken": "Bearer 52d1157f-08fc-4519-8458-82427de17fb7",
-    "resourceId": "WIJZIGER",
```

Comment:
Please revert these changes in this branch, as these are not related to this ticket.